include/aie/Dialect/AIE/Transforms/AIEPathFinder.h

```diff
@@ -79,10 +79,6 @@ using SwitchboxNode = struct SwitchboxNode : SwitchboxNodeBase, Switchbox {
   int id;
 };
 
-// warning: 'xilinx::AIE::ChannelEdge::src' will be initialized after
-// SwitchboxNode &src; [-Wreorder]
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wreorder"
```

Comment:
Personally, I think handling the diagnositcs properly is preferable to having noisy warnings..  That said, I don't understand what this is actually warning about...

Reply:
Down below there's some juggling of llvm/ADT requirements and our own (how a `ChannelEdge` is created most often). I'm the one that put this here and I'm taking it out because I prefer the noisy warning (which is at least precise) to the noisy warning about ~unsupported diagnostics when compiling using clang.